programs/install/Install.cpp

```diff
@@ -856,6 +856,10 @@ namespace
             {
                 fmt::print("The process with pid = {} is running.\n", pid);
             }
+            else
+            {
+                pid = 0;
```

Comment:
Missing comment.

Reply:
What kind? If we can not send 0 signal to the process then the process does not exists anymore, and returning pid of dead process has no sense.